packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.tsx

```diff
@@ -419,7 +427,7 @@ export class ApolloClientBrowserImpl extends ApolloClientClientBaseImpl {
   }
 
   setLink(newLink: ApolloLink) {
-    super.setLink.call(this, ReadFromReadableStreamLink.concat(newLink));
+    super.setLink.call(this, new ReadFromReadableStreamLink().concat(newLink));
```

Comment:
These are classes now because that prints nicer in `console.log` if you need to debug something.